src/Germany/Mews.Fiscalizations.Germany/V2/Model/MultipleTss.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+
+namespace Mews.Fiscalizations.Germany.V2.Model
+{
+    public sealed class MultipleTss
```

Comment:
We don't need this. Make it return a collection of Tss and this class doesn't need to exist.